package cn.day19_Exception.cn.itcast_01.cn.itcast_06;

/**
 * Created by Gypsophila on 2018/8/8.
 * throw:如果出现了异常情况,我们可以吧改异常抛出,这个时候应该排除异常的对象
 *
 */
public class ExceptionDemo {
    public static void main(String[] args) {

    }
}
